File: src/Nethereum.Generators.JavaScript/codeGenNodeTests/MyContractName/DTOs/AllowedOutputDTO.cs
using System;
using System.Threading.Tasks;
using System.Numerics;
using Nethereum.Hex.HexTypes;
using Nethereum.ABI.FunctionEncoding.Attributes;
namespace StandardToken.MyContractName.DTOs
{
    [FunctionOutput]
    public class AllowedOutputDTO
    {
        [Parameter("uint256", "", 1)]
        public BigInteger B {get; set;}
    }
}

File: src/Nethereum.Generators.JavaScript/codeGenNodeTestsVb/MyContractName/DTOs/AllowedOutputDTO.vb
Imports System
Imports System.Threading.Tasks
Imports System.Collections.Generic
Imports System.Numerics
Imports Nethereum.Hex.HexTypes
Imports Nethereum.ABI.FunctionEncoding.Attributes
Namespace StandardToken.MyContractName.DTOs

    <[FunctionOutput]>
    Public Class AllowedOutputDTO
    
        <[Parameter]("uint256", "", 1)>
        Public Property ReturnValue1 As BigInteger
    
    End Class

End Namespace
